main.cpp

```diff
@@ -188,68 +191,89 @@ struct Guitar
 
 };
 
-/*
-Thing 2) wind mill
-5 properties:
-    1) number of blades (unsigned int)
-    2) rotational speed (float)
-    3) pole height (float)
-    4) amount of energy generated in a day (float)
-    5) blade length (unsigned int)
-3 things it can do:
-    1) convert mechanical energy into electrical energy
-    2) rotate their blades
-    3) connect to the electrical grid
- */
+bool Guitar::String::isCoated()
+{
+    return coated;
+}
+
+void Guitar::String::setTension(float localTension)
+{
+    tension = localTension;
+}
+
+float Guitar::String::getGauge()
+{
+    return gauge;
+}
+
+void Guitar::playChord(std::string chord)
+{
+    chordToPlay = chord;
+}
+void Guitar::PlaySolo()
+{
+    // do nothing
+}
+bool Guitar::connectToAmplifier()
+{
+    bool isConnectedToAmp = true;
+    return isConnectedToAmp;
+}
+
 struct WindMill
 {
-    // number of blades
     unsigned int numberOfBlades = 3;
-    // rotational speed
     float rotationalSpeed = 0.0f;
-    // pole height
     float poleHeight = 10.0f;
-    // amount of mechanical energy generated per day
     float mechanicalEnergyGeneratedPerDay = 100.0f;
-    // blade length
     unsigned int bladeLength = 20;
+    bool isConnectedToGrid = false;
 
-    // convert mechanical energy into electrical energy
-    float convertMechEnergyToElEnergy(float mechanicalEnergy = 50.0f); // returns electrical energy
-    // rotate wind mill's blades
-    bool rotateBlades(float rotationalSpeed = 100.0f); // confirm whether or not the blades are rotating
-    // connect to the electrical grid
-    bool connectToGrid(); // confirm whether or not the wind mill is connected to the grid
+    float convertMechEnergyToElEnergy(float mechanicalEnergy = 50.0f);
+    bool rotateBlades(float rotationalSpeed = 100.0f);
+    bool connectToGrid();
 
 };
 
-/*
-Thing 3) Motorcycle
-5 properties:
-    1) RPMs (usigned int)
-    2) color (string)
-    3) size (unsigned int)
-    4) engine type (string)
-    5) brand name (string)
-3 things it can do:
-    1) accelerate
-    2) consume gas, read tank level
-    3) cruise
- */
+float WindMill::convertMechEnergyToElEnergy(float mechanicalEnergy)
+{
+    // Electrical energy is 3.45 times mechanical energy
+    return 3.45f * mechanicalEnergy;
+}
+
+bool WindMill::rotateBlades(float rotSpeed)
+{
+    bool isRotating = false;
+
+    if(rotSpeed > 0.0f)
+    {
+        rotationalSpeed = rotSpeed;
+        isRotating = true;
+    }
+    else
+        rotationalSpeed = 0.0f;
```

Comment:
If you're going to use curly braces for an if statement, use them for the else statement.
A consistent coding style is very important.
- [ ] revise